syntax = "proto3";

option cc_generic_services = true;

import "service/DataService.proto";
package apb.api.dokan;

option java_package = "jp.co.apb.server.api.proto.dokan";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.Dokan";

message RegisterDokanConfirmedRequest {
  repeated int32 dokan_id = 1;
}

message RegisterDokanConfirmedResponse {
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service DokanService {
  rpc RegisterDokanConfirmed(.apb.api.dokan.RegisterDokanConfirmedRequest) returns (.apb.api.dokan.RegisterDokanConfirmedResponse);
}

